lib/RequestHandler.ts

```diff
@@ -0,0 +1,52 @@
+/// <reference path="../typings/tsd.d.ts" />
+import Promise = require('bluebird');
+import restify = require('restify');
+import apiSession = require('./api-session');
+import BAPI = require('./blpapi-wrapper');
+
+export = RequestHandler;
+
+class RequestHandler {
+    private blpSession: BAPI.Session;
+
+    constructor(blpsess: BAPI.Session) {
+        this.blpSession = blpsess;
```

Comment:
I have been reading up on [res.locals](http://expressjs.com/api.html#res.locals) from the [Express](http://expressjs.com/) project and it would be great to have something like this to have some middleware to tack-on `blpapi.Session` for further handlers to use.